ui/src/logic/usePathFromHark.ts

```diff
@@ -0,0 +1,45 @@
+import { udToDec } from '@urbit/api';
+import { useWritByFlagAndWritId } from '@/state/chat';
+import { Skein } from '@/types/hark';
+
+export default function usePathFromHark(bin: Skein, isMention: boolean) {
+  const writId = bin.top?.wer.split('/')[10];
```

Comment:
Oh sorry I see now, I think it would be more clear if this early returned `wer` if it's not a mention above all the rest.